dev/com.ibm.ws.app.manager.springboot.thin/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -0,0 +1,149 @@
+#error.missing.servlet.explanation=The application is a web application, and it requires a Servlet feature to be configured in the server.xml file.
+#error.missing.servlet.useraction=Configure a Servlet feature in the server.xml file.
+
+error.spring2.requires.servlet31.application=CWWKC0254E: The servlet-3.1 or servlet-4.0 feature must be configured. The springBoot-1.5 or springBoot-2.0 feature is configured and the application uses Servlets.
+error.spring2.requires.servlet31.application.explanation=The application uses Servlets but no Servlet feature is configured.
+error.spring2.requires.servlet31.application.useraction=Configure one of the specified Servlet features.
+
+#ADDED for SpringBoot 3.0:
+error.spring3.requires.servlet6.application=CWWKC0274E: The servlet-6.0 feature must be configured. The springBoot-3.0 feature is configured and the application uses servlets.
+error.spring3.requires.servlet6.application.explanation=The application uses Servlets but a no Servlet feature is configured.
+error.spring3.requires.servlet6.application.useraction=Configure the specified Servlet feature.
+
+error.multiple.applications.not.allowed=CWWKC0255E: Spring Boot application {0} cannot be started because application {1} is already active. You cannot configure multiple Spring Boot applications in the same server configuration.
+error.multiple.applications.not.allowed.explanation=Configuring multiple Spring Boot applications in the same server configuration is not supported.
+error.multiple.applications.not.allowed.useraction=Configure only one Spring Boot application per server configuration.
+
+error.no.manifest.found=CWWKC0256E: No META-INF/MANIFEST.MF file found for the Spring Boot application {0}.
+error.no.manifest.found.explanation=Spring Boot applications must contain a META-INF/MANIFEST.MF file.
+error.no.manifest.found.useraction=Configure a valid Spring Boot application, which contains a META-INF/MANIFEST.MF file, with the server.
+
+error.no.spring.class.found=CWWKC0257E: The Spring Boot application META-INF/MANIFEST.MF file must specify a Start-Class header.
+error.no.spring.class.found.explanation=Spring Boot applications must contain a main class specified by the Start-Class manifest header.
+error.no.spring.class.found.useraction=Configure a valid Spring Boot application, which contains a Start-Class manifest header, with the server.
+
+error.missing.ssl=CWWKC0258E: The application failed to start because it requires an SSL feature to be configured in the server.xml file.
+error.missing.ssl.explanation=The application is a secure application, and it requires an SSL feature to be configured in the server.xml file.
+error.missing.ssl.useraction=Configure the transportSecurity-1.0 feature in the server.xml file.
+
+#UPDATED for SpringBoot 3.0:
+#error.missing.websocket=CWWKC0259E: The application failed to start because it requires a websocket feature to be configured in the server.xml file.
```

Comment:
```suggestion
#error.missing.websocket=CWWKC0259E: The application failed to start because it requires a WebSocket feature to be configured in the server.xml file.
```